platform/sun50i/devices.c

```diff
@@ -36,6 +39,16 @@ static struct device msgbox = {
 	.irqdev   = &r_intc,
 };
 
+static struct device pio = {
+	.name     = "pio",
+	.clock    = CCU_GATE(CCU_GATE_PIO) | CCU_RESET(CCU_RESET_PIO),
+	.clockdev = &ccu,
+	.drv      = &sunxi_pio_driver,
+	.drvdata  = BITMASK(1, 7), /*< Physically implemented PIO ports (1-7).
+	                            * */
+	.regs     = DEV_PIO,
```

Comment:
The ordering should match the order of the fields in the struct definition in `dm.h`, so `.regs` should come second, after `.name`. (I recently fixed the other existing devices, which you probably used as examples.)

Reply:
Done.